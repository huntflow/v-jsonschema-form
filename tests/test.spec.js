import { mount } from '@vue/test-utils';
import Vjsf from '../src/components/Form';
import simpleData from '../src/components/Form.stories.data/simple';

test('mount component', async () => {
  expect(Vjsf).toBeTruthy();
  const wrapper = mount(Vjsf, {
    props: {
      ...simpleData
    }
  });
  expect(wrapper.html()).toMatchSnapshot();
});
