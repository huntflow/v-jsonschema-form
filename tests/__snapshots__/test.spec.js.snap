// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`mount component 1`] = `
"<form>
  <!--v-if-->
  <div>
    <fieldset id=\\"root\\">
      <legend id=\\"root__title\\" required=\\"false\\">A registration form</legend>
      <p id=\\"root__description\\" class=\\"field-description\\">A simple form example.</p>
      <div>
        <div>root_firstName TODO: remove later <input type=\\"text\\" autofocus=\\"\\" required=\\"\\" emptyvalue=\\"\\" enumoptions=\\"false\\"></div>
      </div>
      <div>
        <div>root_lastName TODO: remove later <input type=\\"text\\" required=\\"\\" enumoptions=\\"false\\"></div>
      </div>
      <div><input type=\\"number\\"></div>
      <div><textarea id=\\"root_bio\\" class=\\"form-control\\" autofocus=\\"false\\" placeholder=\\"\\" enumoptions=\\"false\\"></textarea></div>
      <div><input type=\\"password\\"></div>
      <div>
        <div>root_telephone TODO: remove later <input type=\\"text\\" inputtype=\\"tel\\" enumoptions=\\"false\\"></div>
      </div>
    </fieldset>
  </div>
</form>"
`;
